client/src/mvc/ui/ui-file-source.js

```diff
@@ -79,7 +79,7 @@ var View = Backbone.View.extend({
 
     /** Returns current value */
     _getValue: function () {
-        return this._value.url;
+        if (this._value) return this._value.url;
```

Comment:
We can use optional chaining to make this type of thing more concise:

```suggestion
        return this._value?.url;
```